backend/game/views/game.py

```diff
@@ -172,3 +173,17 @@ def my_turn(self, request, *args, **kwargs):
         run_player_turn.delay(player.id, serializer.validated_data)
 
         return Response(status=status.HTTP_200_OK)
+
+    @action(detail=True, methods=["get"])
+    def generation(self, request, *args, **kwargs):
+        """View to get the generation field only"""
```

Comment:
Je crois qu'il manque la vérification que l'utilisateur qui souhaite accéder à l'attribut `generation` de la `game` y contrôle bien un `Player`. Et si ce n'est pas la cas il faut renvoyer `raise PermissionDenied(detail="You are not playing in this game")` comme dans `player-full`

Reply:
J'ai hésité en me disant que cette info n'était pas critique d'accès, mais par souci de cohérence avec le reste de l'API tu as raison